costmodel/costmodel.go

```diff
@@ -352,6 +352,9 @@ func ComputeCostData(cli prometheusClient.Client, clientset kubernetes.Interface
 	}
 	currentContainers := make(map[string]v1.Pod)
 	for _, pod := range podlist.Items {
+		if pod.Status.Phase != "Running" {
```

Comment:
Is there potentially a version of k8s that may have a phase that doesn't equal EXACTLY "Running"?